manager/container.go

```diff
@@ -356,7 +372,7 @@ func (self *containerData) nextHousekeeping(lastHousekeeping time.Time) time.Tim
 		}
 	}
 
-	return lastHousekeeping.Add(self.housekeepingInterval)
+	return lastHousekeeping.Add(jitter(self.housekeepingInterval, JitterFactor))
```

Comment:
Isn't it enough to introduce jitter here alone and ceil it at maxHousekeepingInterval?